docker/gaffer/docker-compose.yaml

```diff
@@ -144,15 +144,17 @@ services:
     # accumulo to "auth" with HDFS as the super user so that it can:
     - HADOOP_USER_NAME=hadoop
     volumes:
-    - ../accumulo/conf:${ACCUMULO_CONF_DIR}:ro
+    - ../accumulo/conf-${ACCUMULO_VERSION}:${ACCUMULO_CONF_DIR}:ro
     - /var/log/accumulo
 
   gaffer-rest:
     image: gchq/gaffer-rest:${GAFFER_VERSION}
+    restart: on-failure
```

Comment:
This has been added as sometimes the rest is ready before Accumulo, so the rest errors as it cannot find Accumulo in zookeeper. A more permanent fix would be to implement https://github.com/gchq/gaffer-docker/issues/243